src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/form.html.twig

```diff
@@ -0,0 +1,107 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(cmsPageForm) }}
+<div class="card">
+  <div class="card-header">
+    {{ 'Page'|trans({}, 'Admin.Global') }}
+  </div>
+  <div class="card-block row">
+    <div class="card-text">
+
+      {% set invalidCharsHint = 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ '<>={}' %}
+
+      {{ ps.form_group_row(cmsPageForm.page_category_id, {}, {
+        'label': 'Page category'|trans({}, 'Admin.Design.Feature'),
+      }) }}
+
+      {{ ps.form_group_row(cmsPageForm.title, {}, {
+        'label': 'Title'|trans({}, 'Admin.Global'),
+        'help': invalidCharsHint
+      }) }}
+
+      {{ ps.form_group_row(cmsPageForm.meta_title, {}, {
+        'label': 'Meta title'|trans({}, 'Admin.Global'),
+        'help': invalidCharsHint
+      }) }}
+
+      {{ ps.form_group_row(cmsPageForm.meta_description, {}, {
+        'label': 'Meta description'|trans({}, 'Admin.Global'),
+        'help': invalidCharsHint
+      }) }}
+
+      {{ ps.form_group_row(cmsPageForm.meta_keyword, {}, {
+        'label': 'Meta keywords'|trans({}, 'Admin.Global'),
+        'help': invalidCharsHint
+      }) }}
+
+      {{ ps.form_group_row(cmsPageForm.friendly_url, {}, {
+        'label': 'Friendly URL'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(cmsPageForm.content, {}, {
+        'label': 'Page content'|trans({}, 'Admin.Design.Feature'),
+      }) }}
+
+      {{ ps.form_group_row(cmsPageForm.is_indexed_for_search, {}, {
```

Comment:
I didn't know we now have full macro forms, that's great! ^^
We should consider refacto all the old forms in 177